redwood-lazylayout-uiview/build.gradle

```diff
@@ -25,6 +25,8 @@ kotlin {
     }
     commonTest {
       dependencies {
+        api projects.redwoodLayoutSharedTest
+        api projects.redwoodLayoutUiview
```

Comment:
Unfortunately one of the tests currently assumes `Column`, not an arbitrary flex container.